spec/game_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'game'
+
+describe Game do
+  subject(:game) { described_class.new(player) }
+  let(:player) { double :player }
+
+  describe '#player' do
+    it 'retrieves the player' do
+      expect(game.player).to eq player
+    end
+  end
+
+  describe '#opponent_choice' do
+    it 'returns either Rock, Paper or Scissors' do
+      allow(game).to receive(:opponent_choice) { "ROCK" }
+      expect(game.opponent_choice).to eq "ROCK"
```

Comment:
If you want to test "randomness", you could wrap `["ROCK", "PAPER", "SCISSORS"].sample` inside a method called `random_move`, for example - you would then stub this method to return a specific move (like "ROCK") and then assert the return value of `game.opponent_choice`

This isn't really necessary however, as it doesn't contain any logic (the method is simply returning a value) - but it will become useful once you move the game logic from the views to this class, in order to test than player move v/s a "random" (stubbed) move wins or loses